insertion.py

```diff
@@ -0,0 +1,15 @@
+from main import DataGenerator
+
+
+dg = DataGenerator()
+dg.create_tables()
+dg.df_creation(1000)
+dg.insertion()
+
+"""
+
+ In case you will need to drop tables from database:
+"""
+"""
+dg.drop_tables()
```

Comment:
please, execute this code only within the `__main__` module to prevent from accidental execution on importing that module

Reply:
not resolved yet;
+ kindly asking to move docstrings to the function definition